lib/domodule.js

```diff
@@ -182,7 +185,7 @@ class Domodule {
         const { type: actionType } = Domodule.parseAction(el);
 
         el.removeEventListener(actionType, this.boundActionRouter);
-        el.dataset.domoduleActionProcessed = false;
+        el.dataset.domoduleActionProcessed = 'false';
```

Comment:
you aren't updating conditionals to check strings

Reply:
if tests are passing, we probably need to write a few more to make sure this works